golang/btcspv/types_test.go

```diff
@@ -38,6 +52,45 @@ func TestTypes(t *testing.T) {
 	typesSuite := new(TypesSuite)
 	typesSuite.Fixtures = fixtures
 
+	spvProof := new(SPVProof)
+	err = json.Unmarshal([]byte(typesSuite.Fixtures.Valid[0]), &spvProof)
+	logIfErr(err)
+	typesSuite.Proof = *spvProof
+
+	// TODO: use length of typesSuite.BadHeaders instead of using 5
+	// for i := range typesSuite.BadHeaders {
+	for i := 0; i < 5; i++ {
```

Comment:
What is the best way to write this line using the length of `BadHeaders`? `for i := range typesSuite.BadHeaders` gave an `Index out of range` error.  And `for i := 0; i < typesSuite.BadHeaders.len; i++` gave a `len is not defined` error.